hugegraph-core/src/main/java/com/baidu/hugegraph/StandardHugeGraph.java

```diff
@@ -448,9 +448,16 @@ private BackendStore loadSchemaStore() {
         return this.storeProvider.loadSchemaStore(name);
     }
 
+
     private BackendStore loadGraphStore() {
-        String name = this.configuration.get(CoreOptions.STORE_GRAPH);
-        return this.storeProvider.loadGraphStore(name);
+        String graph = this.configuration.get(CoreOptions.STORE_GRAPH);
+        String backend = this.configuration.get(CoreOptions.BACKEND);
+        if (backend.equals("hbase")) {
+            return this.storeProvider.loadGraphStore(graph, this.configuration);
+        } else {
+            return this.storeProvider.loadGraphStore(graph);
+        }
```

Comment:
can let all backends call `this.storeProvider.loadGraphStore(this.configuration, graph)`